### PR TITLE
new metadb derived table po_line_vendor_reference_number

### PR DESCRIPTION
**Pull request by natalyapik**

### CODE REVIEW

File: sql_metadb/po_line_vendor_reference_number.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-- This derived table extracts refeence numbers from po_lines vendor details.
```
Comment (stdombek) [inline sql_metadb/po_line_vendor_reference_number.sql:1]: Please correct to 'reference'

**Review by stdombek — CHANGES_REQUESTED** [review]
- [ ] Pull request has a complete description
- [X] Query runs without errors
- [X] Query output is correct
- [X] Query logic is clear and well documented
- [X] Query is readable and properly indented with spaces (not tabs)
- [X] Table and column names are in all-lowercase
- [X] Quotation marks are used only where necessary
- [X] Query begins with user documentation in comment lines
- [X] File name is listed in `runlist.txt` after dependencies
- [X] All columns have indexes
- [X] Table is vacuumed and analyzed

Very good! 

- Please make the correction in the description. (line 1) 
- The indentation in the From Clause could be done at the same time. I don't see them that strictly now.
- Unfortunately, a description of the PR is missing.

**Review by stdombek — APPROVED** [review]
PR description is missing, but can now be included in the CHANGES.md. Therefore, the PR can be approved.

**Review by scotttperry — APPROVED** [review]
Pull request has a complete description
 Query runs without errors
 Query output is correct
 Query logic is clear and well documented
 Query is readable and properly indented with spaces (not tabs)
 Table and column names are in all-lowercase
 Quotation marks are used only where necessary
 Query begins with user documentation in comment lines
 File name is listed in runlist.txt after dependencies
 All columns have indexes
 Table is vacuumed and analyzed

**Comment by nassibnassar** [comment]
The SQL file is in the wrong directory.  I will fix it before merging.

**Pull request merged**